### ISSUE TITLE
Error while running demo.py

### ISSUE THREAD

**Issue by sangeun99** [harlanhong/CVPR2022-DaGAN#68]
Hello
Thanks for your code

I am running demo.py now but i faced " RuntimeError: PytorchStreamReader failed reading file data/2: invalid header or archive is corrupted " code while doing.

My shell code was like this.

`CUDA_VISIBLE_DEVICES=0 python demo.py  \
--config config/vox-adv-256.yaml  \
--driving_video path-video  \
--source_image path-img \
--checkpoint modules/depth/models/DaGAN_vox_adv_256.pth.tar \
--relative \
--adapt_scale \
--kp_num 15 \
--generator DepthAwareGenerator`

And the error code was like this
![image](https://user-images.githubusercontent.com/63828057/236667715-b9937ec3-f14e-4bde-b370-26532c6a5e20.png)

I downloaded your pre-trained model from one-drive.
Is there any problem with checkpoint file (DaGAN_vox_adv_256.pth.tar) or just the problem with my env?

AND one more question for evaluating model!
I want to evaluate the model
Do you have any codes to make mp4s for testing? Could you provide it?

Thanks in advance

**Comment by sangeun99** [comment]
@harlanhong
Thanks for your prompt reply.
I figured it out that it was my fault while uploading the file!

Just more questions for evaluation!
To check SSIM or PSNR, is the ground truth driving video?
and do you have any codes to make mp4s for testing? Could you provide it?

Thanks a lot

**Comment by harlanhong** [comment]
Yes, for the evaluation, the driving videos are taken as the ground truth for same-identity experiments. For the evaluation codes, I will arrange them after these busy days. Actually, you can test them frame by frame.

**Comment by sangeun99** [comment]
@harlanhong thanks for your help! I want to try evaluation for myself, is it okay to set source image as the first frame of the driving video and evaluate it?
